xbmc/windowing/WinSystem.h

```diff
@@ -173,6 +173,10 @@ class CWinSystemBase
 
   virtual std::vector<std::string> GetConnectedOutputs() { return {}; }
 
+  virtual bool DisplaySupportsHLG() { return false; }
```

Comment:
CWinSystem seems to be getting more and more bloated by HDR stuff.is it really necessary to add three methods purely for indication? Do you think we could better handle thing in our current HDR handling?

Reply:
> is it really necessary to add three methods purely for indication?

I think the idea behind quietvoid is to use these methods in the future to handle DV streams in the right way. Some extensions / codec overrides need to be added but only on systems that support DV. Otherwise fallback HDR10 compatibility is broken. That is, some streams are HDR10 + DV compatible and the codec to be used is selected based on the capabilities of system (or the current device/firmware settings).

I also suggested something like this in another PR: https://github.com/xbmc/xbmc/pull/19099#issuecomment-950202096

> Do you think we could better handle thing in our current HDR handling?

Indeed, there are many different ways to implement this. For example we could use a single method and return a structure with the supported HDR types. But this required touching the code that already exists with the risk of introducing errors. Since this is a "work in progress" I am fine with introducing new methods on the fly. Perhaps later it will be seen more clearly if it can be re-organized / unified in a better way.

Of course this is just my opinion. @irusak, I know that you are or were working on HDR Linux / LibreELEC things so I wanted you to participate in this PR and you can see things that I do not see or that I am not even aware of.

Reply:
The idea here is simply to have access to the HDR capabilities globally, and I just ended up on `CWinSystem` as a convenient class as it already used `CAndroidUtils::IsHDRDisplay()`.

An alternative could just be a list of a new enum type for all the common HDR formats, and some helper method to avoid boilerplate everywhere.

Reply:
I created a class as an alternative to the different functions in `CWinSystem`.
Do let me know if it's a viable improvement over the previous idea.